x-pack/plugins/reporting/server/lib/esqueue/__tests__/fixtures/elasticsearch.js

```diff
@@ -3,105 +3,108 @@ import elasticsearch from 'elasticsearch';
 import { constants } from '../../constants';
 
 export function ClientMock() {
-  this.indices = {
-    create: () => Promise.resolve({ acknowledged: true }),
-    exists: () => Promise.resolve(false),
-    refresh: () => Promise.resolve(),
-  };
+  this.callWithInternalUser = (ep, params = {}, ...rest) => {
```

Comment:
**nit** I stuck for a moment thinking about what `ep` is. shouldn't we name it as `endpoint`?